wasm/test_page/bergamot.html

```diff
@@ -125,6 +127,7 @@
 quiet: true
 quiet-translation: true
 gemm-precision: int8shift
+alignment: true
```

Comment:
This option is of no relevance to this PR, at least as of now. Sentence alignments are available irrespective of this option. Actually, legal values are "hard", "soft"  or empty where there is no alignments I think. This is a marian-option, so follow docs at https://marian-nmt.github.io/docs/cmd/marian-decoder/

```
--alignment TEXT                      Return word alignment. Possible values: 0.0-1.0, hard, soft
```

The option indicates marian-backend should compute alignments during model construction. However, the Response will have (word) alignments only if explicitly indicated through `ResponseOptions`.

Reply:
You are right. I am removing alignment option from the config here.